src/components/List/ListItemHeader.jsx

```diff
@@ -129,17 +130,23 @@ class MessageConversationListItem extends Component {
     }
 }
 
+MessageConversationListItem.propTypes = {
+    checkedIds: propTypes.array,
+    children: propTypes.array,
```

Comment:
Shouldn't children generally be `propTypes.node`?

Reply:
Yeah, that was my first guess as well. But that resulted in a props error. I think this is because this app was written before `React.Fragment` was introduced. Back then, if you didn't want to introduce a wrapper element, your only options was the array notation.